### PR TITLE
Grammar has been done

### PR DESCRIPTION
**Pull request by LiudmilaZhdanok**

### CODE REVIEW

**Review by DenisAbramovDm — DISMISSED** [review]

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Is Your mum working today?
+Why are this boys talking now?
+Where does he live?
+Does Their mother cook very well?
+Lili is not rich, she doesn't drive a Mercedes.
+Let´s go out. It isn't raining now.
+I have to go now. It's getting dark
+A young man usually standes at the window.
```
[the remark this answers is not in the document]
Comment (LiudmilaZhdanok) [reply homework/english/zhdanok_liudmila/lecture1/task1.md]: ok

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Ann usually makes doll dresses herself.
+Does His wife have coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys are comeing home.
```
[the remark this answers is not in the document]
Comment (LiudmilaZhdanok) [reply homework/english/zhdanok_liudmila/lecture1/task1.md]: ok

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Does Maila like Chinese food?
+My grandmother does not work. She is on pension.
+What is your brother watching now?
+Where do you live?
+Do You  play the piano well?
+Where is their brother swiming now?
```
[the remark this answers is not in the document]
Comment (LiudmilaZhdanok) [reply homework/english/zhdanok_liudmila/lecture1/task1.md]: ok

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+What do they eat at breakfast?
+Are They swiming at the moment?
```
[the remark this answers is not in the document]
Comment (LiudmilaZhdanok) [reply homework/english/zhdanok_liudmila/lecture1/task1.md]: ok

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Is Your mum working today?
+Why are this boys talking now?
+Where does he live?
+Does Their mother cook very well?
+Lili is not rich, she doesn't drive a Mercedes.
+Let´s go out. It isn't raining now.
+I have to go now. It's getting dark
+A young man usually stands at the window.
+Ann usually makes doll dresses herself.
+Does His wife have coffee at the moment?
```
Comment (olgaz-skiba) [inline homework/english/zhdanok_liudmila/lecture1/task1.md:14]: at the moment - present continuous?

File: homework/english/zhdanok_liudmila/lecture1/task1.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Look! The boys are coming home.
+Does Maila like Chinese food?
+My grandmother does not work. She is on pension.
+What is your brother watching now?
+Where do you live?
+Do You  play the piano well?
+Where is their brother swimming now?
+What do they eat at breakfast?
+Are They swimming at the moment?
+Mike doesn't wear a tie today.
```
Comment (olgaz-skiba) [inline homework/english/zhdanok_liudmila/lecture1/task1.md:25]: today - marker PresentContinuous?

**Review by olgaz-skiba — APPROVED** [review]

**Review by DenisAbramovDm — APPROVED** [review]

**Pull request merged**